.vscode/launch.json

```diff
@@ -207,6 +207,7 @@
                 "VSC_FORCE_REAL_JUPYTER": "true", // Enalbe tests that require Jupyter.
                 "VSC_JUPYTER_CI_RUN_NON_PYTHON_NB_TEST": "", // Initialize this to run tests again Julia & other kernels.
                 "VSC_JUPYTER_RUN_NB_TEST": "true", // Initialize this to run notebook tests (must be using VSC Insiders).
+                "VSC_JUPYTER_WEBVIEW_TEST_MIDDLEWARE": "true", // Initialize to create the webview test middleware

```

Comment:
All an env var to force on the test middle ware, needed for the middleware VariablesComplete message, and handy to be able to turn on quick just when running VS Code from debugger.